gwdetchar/lasso/tests/test_core.py

```diff
@@ -51,18 +59,35 @@
 
 # -- unit tests ---------------------------------------------------------------
 
-def test_find_outliers():
```

Comment:
I suggest keeping this unit test name the same, though it is a good idea to add the commend line below. Line 64 of this diff, which reads `# find expected outliers`, can be deleted in that case